WFInfo/Ocr.cs

```diff
@@ -918,7 +918,7 @@ private static List<InventoryItem> FindAllParts(Bitmap filteredImage)
                                             continue;
                                         }
                                     }
-                                    else if (currentWord.Length < 2)
+                                    else if (currentWord.Length < 2 && Settings.locale == "en")
```

Comment:
Add a minimum Korean word length. Note the OCR tends to only highlight one or two charters in Korean.